stories/features/welcome/page/fakeLocale.ts

```diff
@@ -13,8 +13,11 @@ const locale = {
   import: 'Import',
   imported: 'Imported',
   // third screen
-  setDefaultSearchEngine: 'Change your search engine',
-  chooseSearchEngine: 'Choose the search engine you would like to use by default when searching the web from the address bar.',
+  setDefaultSearchEngine: 'Set default search engine',
+  chooseSearchEngine: 'Choose the search engine you would like to use by default when searching the web from the URL bar.',
```

Comment:
Using string taken from linked ticket from brave-browser, need to double check which is more current